src/index.ts

```diff
@@ -34,3 +34,5 @@ export * from "./switch/switch.module";
 export * from "./table/table.module";
 export * from "./tabs/tabs.module";
 export * from "./tiles/tiles.module";
+// experimental ... to be removed
```

Comment:
Let's keep it :)
Could be useful for future updates. Experimental features can then be always available to our users.
```suggestion
```

Reply:
Agreed!